src/docs/binding-properties.md

```diff
@@ -0,0 +1,114 @@
+# Properties
+
+## Simple Properties on Swift Structs
+
+Properties in Swift structs have different calling conventions depending on whether the struct is frozen or not, and whether it's a getter or setter:
+
+- Getters:
+  - For frozen structs: Take the struct value lowered into registers
+  - For non-frozen structs: Take a pointer to the struct instance
+  
+- Setters:
+  - For both frozen and non-frozen structs: Always take a pointer to self and a value to set
+  - The pointer to self is required because setters need to modify the struct's memory in-place
```

Comment:
Is this true for all mutating functions including setters?

Reply:
Thats a great point! Yes, this will be true for all mutating functions. Setters would be just one of the use cases